hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml

```diff
@@ -67,6 +67,9 @@
                                         <field name="date"/>
                                         <field name="project_id"/>
                                         <field name="task_id"/>
+                                        <field name="company_id"/>
+                                        <field name="employee_id"/>
+                                        <field name="new_line_id" invisible="1"/>
```

Comment:
`new_line_id` it's needed here, if not, that bug is not fixed. I checked it. And I would say `company_id` and `employee_id` also should be here...

Reply:
`company_id` is needed only for multi-company, right? Maybe we should limit it to multi company group?

Reply:
Re: `new_line_id` - please add a comment about that, to avoid it being removed by some migrator in future

Reply:
@alexey-pelykh done

Reply:
Why `company_id` and `employee_id` are needed in the view? Just in case?

Reply:
Yeah, because it's not a tree view in a sense. The widget fetch the data from those fields. If we not put this fields in this tree, then this fields in the lines will not have values. It's just the widget works this way.